[2022-08-29 17:12:49,968] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: h1b_lca_data_v3.entry_task scheduled__2021-04-10T06:00:00+00:00 [queued]>
[2022-08-29 17:12:49,984] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: h1b_lca_data_v3.entry_task scheduled__2021-04-10T06:00:00+00:00 [queued]>
[2022-08-29 17:12:49,986] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-08-29 17:12:49,987] {taskinstance.py:1239} INFO - Starting attempt 61 of 61
[2022-08-29 17:12:49,988] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-08-29 17:12:50,007] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): entry_task> on 2021-04-10 06:00:00+00:00
[2022-08-29 17:12:50,017] {standard_task_runner.py:52} INFO - Started process 1224 to run task
[2022-08-29 17:12:50,023] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'h1b_lca_data_v3', 'entry_task', 'scheduled__2021-04-10T06:00:00+00:00', '--job-id', '1610', '--raw', '--subdir', 'DAGS_FOLDER/upload_h1b_data_to_gcp.py', '--cfg-path', '/tmp/tmptkjow7xn', '--error-file', '/tmp/tmp6w6nqk66']
[2022-08-29 17:12:50,026] {standard_task_runner.py:77} INFO - Job 1610: Subtask entry_task
[2022-08-29 17:12:50,123] {logging_mixin.py:109} INFO - Running <TaskInstance: h1b_lca_data_v3.entry_task scheduled__2021-04-10T06:00:00+00:00 [running]> on host f153c6531160
[2022-08-29 17:12:50,199] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=h1b_lca_data_v3
AIRFLOW_CTX_TASK_ID=entry_task
AIRFLOW_CTX_EXECUTION_DATE=2021-04-10T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-10T06:00:00+00:00
[2022-08-29 17:12:50,202] {python.py:175} INFO - Done. Returned value was: None
[2022-08-29 17:12:50,234] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=h1b_lca_data_v3, task_id=entry_task, execution_date=20210410T060000, start_date=20220829T171249, end_date=20220829T171250
[2022-08-29 17:12:50,280] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-08-29 17:12:50,384] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
